content/member-handbook.md

```diff
@@ -147,6 +147,186 @@ That said, here are some things we might do, in rough order of severity:
   Usually used when someone refuses to step back from a heated conversation or for trolls.
 - **ban**:  asking someone to log off completely or doing the same via admin tools.
 
+## Job Postings
+
+The `#jobs` channel is a space that we've dedicated for people to post when they
+are looking for work, or when they have jobs.  This section will provide some
+guidance and rules for posting in the `#jobs` channel.
+
+### Pinning Posting Durations
+
+#### About Pinning
```

Comment:
pinning is like the least important part of all of this. It should go at the end.